docs/development/pigs/pig-024.rst

```diff
@@ -55,50 +55,49 @@ release).
-
-Any contribution without threshold on these topics renders a user as official Gammapy contributor. Many of these
-contributions are tracked using the git history. However, the use of personal data is regulated in Europe and one
-should have the authorisation of users to be cited with their personal information for publications (their full name
-is mandatory, and, if any, their affiliation and their ORCID number). For this reason, we propose to use the
-`Developer Certificate of Origin` (DCO, see the text in `here <https://developercertificate.org/>`_) for each commit.
-Its acceptation by contributors will permit to:
-
-- certify that a user wrote or otherwise has the right to submit a code under our open source licence,
-- allow our project to use their personal data for the contributions' record (ie for releases).
-
-The file ``README.md`` will contain a `Contributing` section explaining that the project follows this DCO that will be
+* contributing to gammapy source code,
+* contributing to its documentation (rst pages, docstring, gammapy-webpage),
+* contributing to code reviews, maintenance, deployment and DeVops,
+* contributing to our associated projects (gammapy-benchmarks, etc),
+* organizing communication, e.g. hands-on sessions, schools, social media,
+* coordinating the project.
+
+Contributions on any of the aforementioned aspects make a user as official Gammapy contributor.
+
+Many of these contributions are tracked using the git history. However, the use of personal data is
+regulated in Europe and one should have the authorisation of users to be cited with their personal
+information for publications (their full name is mandatory, and, if any, their affiliation and
+their ORCID number). For this reason, we propose to use the *Developer Certificate of Origin*
+(DCO, see the text in `here <https://developercertificate.org/>`_) for each commit. Its
+acceptation by contributors will permit to:
+
+* certify that a user wrote or otherwise has the right to submit a code under our open source licence,
+* allow our project to use their personal data for the contributions' record (ie for releases).
+
+The file ``README.md`` will contain a *Contributing* section explaining that the project follows this DCO that will be
 given in a new ``CONTRIBUTING.md`` file, as well as a link to this PIG.
 
 .. important::
-    For practical reasons, the developers strongly recommend that the contributors use always the same GitHub account
+    For practical reasons, it is strongly advised that the contributors use always the same GitHub account
     with a valid full name, to synchronise their git email addresses with their GitHub addresses, to have a unique
     `ORCID <https://orcid.org/>`_ identifier and store it into their GitHub user profile for commodity, and inform the
     developers of any change of affiliation or email address.
 
 
-Definition of the `Maintainer`
+Definition of the *Maintainer*
```

Comment:
Why *the* ? We have multiple maintainers, right?

I'd just leave out the article (for both Contributor and Maintainer):

> Definition of Maintainer

Reply:
I also support consistency here, I think either `Definition of Maintainer` or `Definition of a Maintainer` (consistent with the contributor section right now) would be fine by me.